components/user-list/UserList.js

```diff
@@ -32,16 +32,45 @@ export default class UserList extends Component {
           products: this.state.products,
         })
       }}
-    />
-  )
+    />)
```

Comment:
Acho que não ficar legal recuar o parenteses, que que tem uma função logo em seguida 
```suggestion
    />
  )
```